cli/src/main/java/keywhiz/cli/commands/RenameAction.java

```diff
@@ -16,26 +16,24 @@ public RenameAction(RenameActionConfig config, KeywhizClient keywhiz) {
   }
 
   @Override public void run() {
-    validate();
+    switch (config.resourceType) {
```

Comment:
I removed support for ID-based rename, it looks like almost everything else in the CLI uses names (good for humans), which means we no longer have the exclusive-or validation case, which in turn means we can now rely on JCommander's built-in validation now that all parameters are required.

Reply:
nice!